syntax = "proto3";

package private_join_and_compute;

message PolynomialMessageRoundTwo {
  repeated uint64 polynomial_input_0 = 1;
  repeated uint64 polynomial_input_1 = 2;
};